platform/backend/api/src/integrationTest/kotlin/io/hamal/api/http/controller/trigger/TriggerStatusControllerTest.kt

```diff
@@ -0,0 +1,103 @@
+package io.hamal.api.http.controller.trigger
+
+import io.hamal.lib.domain._enum.TriggerStatus.Active
+import io.hamal.lib.domain._enum.TriggerStatus.Inactive
+import io.hamal.lib.domain._enum.TriggerType
+import io.hamal.lib.domain.vo.*
+import io.hamal.lib.http.HttpErrorResponse
+import io.hamal.lib.http.HttpStatusCode
+import io.hamal.lib.http.HttpSuccessResponse
+import io.hamal.lib.http.body
+import io.hamal.lib.sdk.api.ApiError
+import io.hamal.lib.sdk.api.ApiTriggerCreateReq
+import io.hamal.lib.sdk.api.ApiTriggerCreateSubmitted
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.Test
+import kotlin.time.Duration.Companion.seconds
+
+internal class TriggerStatusControllerTest : TriggerBaseControllerTest() {
+
+    @Test
+    fun `Default status is active`() {
+        val triggerId = createFixedTrigger()
+        with(getTrigger(triggerId)) {
+            assertThat(id, Matchers.equalTo(triggerId))
```

Comment:
static imports to increase readability
```suggestion
            assertThat(id, equalTo(triggerId))
```